terraform/service_export.tf

```diff
@@ -138,6 +138,7 @@ resource "google_cloud_run_service_iam_member" "export-invoker" {
 
 resource "google_cloud_scheduler_job" "export-worker" {
   name             = "export-worker"
+  region           = local.appengine_region
```

Comment:
This doesn't always work. For example, the GAE region is `us-central`, but the corresponding Cloud Scheduler region is `us-central1` (note "1" at the end).